policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/childrens_fitness_tax_credit/base.yaml

```
@@ -0,0 +1,9 @@
+description: Yukon child fitness fee $1,000 for eligible fees paid for the cost of registration or membership.
+values:
+  2022-01-01: 1_000
+metadata:
+  unit: currency-CAD
+  label: Yukon child fitness base fee
```

Isn't this closer to what it does? And let's include the program name in the labels
```suggestion
  label: Yukon children's fitness tax credit maximum amount
```